config/jobs/kubernetes/sig-release/release-branch-jobs/1.26.yaml

```diff
@@ -326,7 +326,7 @@ periodics:
   - 'perfDashJobType: performance'
   - 'perfDashBuildsCount: 500'
 - annotations:
-    fork-per-release-periodic-interval: 24h
+    fork-per-release-periodic-interval: ""
```

Comment:
This is wrong, there must be one interval left here or we're going to run into issues the next time we rotate jobs.